client/js/components/App.js

```diff
@@ -27,10 +28,11 @@ class App extends Component {
     const { currentUser, actions } = this.props;
 
     return (
-      <div className="col-md-10 col-md-push-1">
-        <Header user={currentUser} onLogout={actions.logout} onLogoutSuccess={this.redirectToHomePage.bind(this)}/>
+      <section>
```

Comment:
yes, changes I've made was for consistency sake. if the consistency means one way or another is other story, we should discuss.